src/main.rs

```diff
@@ -40,8 +40,11 @@ impl ByteSet {
             | (ByteSet::Excludes(excluded), ByteSet::Includes(included)) => {
                 HashSet::is_subset(included, excluded)
             }
-            // Too hard to enumerate all of these: just say that they aren’t disjoint
-            (ByteSet::Excludes(_), ByteSet::Excludes(_)) => false,
+            // Hard: enumerate these by brute force - they are disjoint if all
+            // bytes are contained exclusively in one byte set or the other set
```

Comment:
Does it need to be exclusive? I think they can be in both, they just have to be in at least one :thinking:

Reply:
Right, in order for two bytesets to be disjoint there must be one byte in both sets, which means there must be one byte that isn't excluded by _either_ set, but a byte excluded by _both_ sets is fine.